modules/kubernetes-addons/local-volume-provisioner/local-static-provisioner/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: provisioner
+description: local provisioner chart
```

Comment:
We should reference the existing chart and not take on re-creating/copying the chart here https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm/provisioner

Reply:
Hello @bryantbiggs, there is no helm repo for this chart.
https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/199

I am not sure if setting 
`repository = git+https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner.git@helm?ref=master` will work with the provider 

I will give a try and see the outcome.

Reply:
@bryantbiggs what do you think about merging it as is and when the repo will be available we can switch to it. I do post a comment here https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/289#issuecomment-1240497382 to fix the helm repo for that chart , I will follow up but in the meantime can we keep it like this?